frontend/src/employee-frontend/components/unit/unit-reservations/UnitAttendanceReservationsView.tsx

```diff
@@ -161,17 +161,32 @@ export default React.memo(function UnitAttendanceReservationsView({
               saveExternalAttendance={saveExternalAttendance}
             />
           ) : (
-            <ChildReservationsTable
-              unitId={unitId}
-              operationalDays={childData.operationalDays}
-              allDayRows={
-                childData.groups.find((g) => g.group.id === groupId)
-                  ?.children ?? childData.ungrouped
-              }
-              onMakeReservationForChild={setCreatingReservationChild}
-              selectedDate={selectedDate}
-              reloadReservations={reloadChildReservations}
-            />
+            <>
+              <StaffAttendanceTable
+                operationalDays={childData.operationalDays}
+                staffAttendances={staffData.staff.filter((s) =>
+                  daycareAclRows
+                    .find((r) => r.employee.id === s.employeeId)
+                    ?.groupIds.includes(groupId)
+                )}
```

Comment:
Perhaps not a thing to fix in this PR, but these pretty complex `filter`/`find`/`includes` combos could perhaps be moved to the component top level by employing `combine` and `Result.map`. That way they would also only run once when the `Result` changes instead of on every render.